src/components/HomeSection.jsx

```diff
@@ -5,61 +5,62 @@ import { Container, Grid, Typography } from '@mui/material';
 const HomeSection = ({
   alignItems,
   direction,
-  tab,
   title,
+  main,
```

Comment:
used to change typography and image size. I noticed on the figma design that the top section had bigger text and a bigger image.

Reply:
What if we just pass in the whole `img` node instead?

Reply:
Oh nvm, you also need to make the typography a little bigger too.